### DISCUSSION TITLE
AI智能書桌

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#596] [General]
我覺得這就跟小學時候，那種超多功能的方形鉛筆盒一樣炫泡，什麼削鉛筆機啦，超多內層之類的，各種意義上是非常之實用的，擁有那個鉛筆盒你在班上的地位肯定不低。反觀這張智能書桌，我倒不覺得有什麼厲害之處，我寧願買張電腦升降桌還比較酷，總之給大家參考看看，或許有不同意見
https://world.taobao.com/item/692965494794.htm?spm=a21wu.10013406-20221201-1-tw.taglist-content.1.2078677c3jo5bq

**Comment by awehao** [comment]
這還挺新穎的诶，歲然感覺很多功能很多於，但在念書休息之餘，把書桌當玩具玩，也是一大樂趣，隨著時代在變，各種AI知能的家電，家具都不斷再翻新，在我們享受這些方便的同時，也要懂得知足常樂，前人種樹後人乘涼

**Comment by bww0401** [comment]
我覺得它最棒的功能在於它可以偵測姿勢的正確或錯誤，來挑整高度。應該大家都有類似的經驗，就是當寫字很專注的時候，很容易脖子會越彎越低，如果使用這個產品，感覺可以有效避免駝背!

**Comment by manipulateed** [comment]
會不會之後隨便什麼產品都能冠上AI之名，實際上卻沒有什麼幫助，如果要監測什麼姿勢或是專心程度，也不一定要侷限在書桌上吧，反而獨立出一個監測各種時刻的設備比較有效率。

**Comment by awkward-willy** [comment]
單論這項產品可能確實沒有許多實質幫助，但若能讓家中的各項產品都佐以資訊科技，才能發揮完整的價值，且形成完整的生態鏈。

**Comment by Raywei0214** [comment]
但我覺得要搞這種居家AI生態鏈反而不太實際，成本太高，你可能要花一大筆錢，但你的生活上可能就只是少走個一兩步去拿冷氣遙控器
，我可能寧願花這筆錢升級我的電腦還可以享有更優質的遊戲體驗之類的

**Comment by larry40305** [comment]
我覺得這產品以後就會出現家長遠程監控的功能，我覺得亞洲家長會非常需要這個功能。
